features/step_definitions/star_sign_steps.rb

```diff
@@ -0,0 +1,20 @@
+Given(/^it has been created$/) do
+  @customer = Customer.create!(username: "joe", 
+    email: "joe@bloggs.com",
+    password: "password", 
+    password_confirmation: "password",
+    confirmation_sent_at: Time.now - 1.day,
+    confirmed_at: Time.now,
+    date_of_birth: Date.new(1966,2,12))
+
+  @profile = Profile.create!(
+    biography: "A long time ago, in a galaxy far away...", 
+    customer: @customer)
+end
+
+Then(/^their star sign is automatically set$/) do
+  star_signs = ["Capricorn","Aquarius","Pisces","Aries","Taurus","Gemini",
```

Comment:
My thought is that this is a fairly minor bit of duplication that I can't see being used elsewhere